_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,107 @@
+ownership of these low risk procurements to the lines of business and simplify the process. In their [Procurement in 2020 and Beyond](https://www.gartner.com/en/procurement-operations/
+trends/procurement-in-2020?utm_expid=.bUMMSUQqSDOvsM8TqeyBQw.0&utm_referrer=https%3A%2F%2Fwww.google.com%2F) report, Gartner recommends 5 shifts regarding digital procurement. One of
+these involves shifting the ownership for low and mid-tier purchases away from corporate procurement teams so that they can focus primarily on strategic, high value, high risk purchases.
+To ensure lines of business feel confident in taking this on, the process should be simplified and automated as much as possible. Simplifying GC procurement processes has long been
+identified as a need including the recommendation from the red tape reduction report of establishing “a fast track process for service contracts under $10,000”.
+
+As part of an effort to explore significant and recurring issues in federal procurement, the Office of the Procurement Ombudsman released a report on [Low Dollar Value Contracting](http:/
+/opo-boa.gc.ca/documents/faiblevaleur-dollarvalue-eng.pdf) in 2019.  In this report they advised that “The regulatory and policy framework for federal contracting allows federal
+organizations the flexibility to develop their own frameworks to govern LDV contracting.”  Conversations with the folks working on the [procurement policy reset](https://www.gcpedia.gc.
+ca/wiki/Procurement_Policy_Division,_Treasury_Board_of_Canada_Secretariat/Policy_Suite_Reset) (internal GC link) at TBS have also confirmed that there is nothing in the current policy
+instruments that prevent establishing a simplified process for LDV purchases.
+
+While having a simplified process for low dollar value procurement would be good for the GC in general, it would be especially useful for IT. IT modernization depends on procurement
+modernization. One can't move forward without the other. We can't work in an agile or DevOps way if procurement processes still follow waterfall. The Digital Operations Strategic Plan
+highlights this, “If modern technology is an essential enabler for digital government, procurement modernization is an equally essential enabler of modern technology.”  Equally, we need
+to think about how our procurement processes help or hinder the uptake of open source in the GC.
+
+Instead of the example above regarding finding and fixing a bug, imagine that you need a small IT solution.  You find an open source application that would meet your needs but it does
+not meet GC accessibility requirements. What if you could quickly and easily pay someone to fix the accessibility issues so that you could choose that open source solution and move
+forward without delay?  If this existed would you be more likely to consider using open source solutions?
```

Comment:
Could add a plug of the digital standards here ... 

If this existed, would you be more likely to consider using open source solutions (in line with the GC Digital Standards, as discussed below)?